site2/docs/client-libraries-cpp.md

```diff
@@ -341,30 +341,24 @@ ${PULSAR_HOME}/pulsar-client-cpp/build/lib/Release/pulsar.dll
 
 ## Connection URLs
 
-To connect Pulsar using client libraries, you need to specify a Pulsar protocol URL.
+To connect to Pulsar using client libraries, you need to specify a [Pulsar protocol](developing-binary-protocol.md) URL.
 
-Pulsar protocol URLs are assigned to specific clusters, you can use the Pulsar URI scheme. The default port is `6650`. The following is an example for localhost.
+You can assign Pulsar protocol URLs to specific clusters and use the `pulsar` scheme. The following is an example of `localhost` with the default port `6650`:
 
 ```http
-
 pulsar://localhost:6650
-
 ```
 
-In a Pulsar cluster in production, the URL looks as follows. 
+If you have multiple brokers, separate `ip:port` by commas:
 
 ```http
-
-pulsar://pulsar.us-west.example.com:6650
-
+pulsar://localhost:6550,localhost:6651,localhost:6652
 ```
 
-If you use TLS authentication, you need to add `ssl`, and the default port is `6651`. The following is an example.
+If you use [TLS](security-tls-authentication.md) authentication, add `+ssl` in scheme:
```

Comment:
```suggestion
If you use [TLS](security-tls-authentication.md) authentication, add `+ssl` in the scheme:
```
It needs a definite article since it refers to the `pulsar` schema mentioned in line346. 
5 more occurrences need an update too.

Reply:
Updated. It's possible we find a place to describe all about the connection URLs format and leave only the simplest `pulsar://localhost:6650` in each client libraries document.

Reply:
Perhaps `developing-binary-protocol` or I will regard it as a basic concept. This can be a follow-up improvement.